extensions/search-rubygems/src/components/GemOptions.tsx

```diff
@@ -0,0 +1,62 @@
+import { Icon, PushAction, List, ActionPanel, OpenInBrowserAction, CopyToClipboardAction } from "@raycast/api";
+import type { GemSearchResult } from '../rubygems/types';
+import { GemInfo } from './GemInfo';
+import { mapGemLinks } from '../utils';
+
+interface Props { gem: GemSearchResult };
+
+export const GemOptions = ({ gem }: Props): JSX.Element => {
+  return (
+    <List navigationTitle={`Search RubyGems > ${gem.name}`}>
+      <List.Section title="Actions">
+        <List.Item
+          key="gem-name"
+          icon={Icon.ArrowRight}
+          title="Show details"
+          accessoryTitle="Dependencies and more"
+          actions={
+            <ActionPanel>
+              <PushAction
```

Comment:
No icon added, please add an icon for PushAction

Reply:
@pernielsentikaer fixed!